### PR TITLE
DTDQM Fix to testUnit

### PR DESCRIPTION
**Pull request by jfernan2**
#### PR description:
This PR fixes IB crash reported on:
https://cmssdt.cern.ch/SDT/cgi-bin/logreader/slc7_amd64_gcc900/CMSSW_12_0_X_2021-06-16-2300/unitTestLogs/DQM/Integration#/
Produced by changes on https://github.com/cms-sw/cmssw/pull/34153

PR is used to replace colon by semicolon in python files of the packages

### CODE REVIEW

**Comment by jfernan2** [comment]
please test

**Comment by smuzaffar** [comment]
test parameters:
- addpkg = DQM/Integration

**Comment by smuzaffar** [comment]
please test
@jfernan2 , I am restarting the test just to make sure that `DQM/Integration` is checked out and run the failing unit tests

**Comment by Dr15Jones** [comment]
Why the change from single quotes to double quotes? In python those are completely equivalent.

File: DQM/DTMonitorModule/python/dt_dqm_sourceclient_common_cff.py [hunk 1]
```diff
@@ -104,7 +104,7 @@
 # Local Trigger task for test pulses
 from DQM.DTMonitorModule.dtTriggerTask_TP_cfi import *
 from DQM.DTMonitorClient.dtLocalTriggerTest_TP_cfi import *
-dtTPTriggerTest.hwSources = 'TM'
+dtTPTriggerTest.hwSources = ['TM']
```
Comment (Dr15Jones) [inline DQM/DTMonitorModule/python/dt_dqm_sourceclient_common_cff.py:107]: The two changes in this file are all that is needed to fix the unit tests.

**Comment by jfernan2** [comment]
> Why the change from single quotes to double quotes? In python those are completely equivalent.

This has been a misunderstanding from my side sorry... since they are equivalent I suggest taking the PR as it is

**Comment by jfernan2** [comment]
+1

**Comment by qliphy** [comment]
+1

**Pull request merged**